vtexdocs/RapiDoc
# Feat/postman collection

#### What is the purpose of this pull request?

- To add the option to view/download the corresponding postman collection file from a openAPI schema in the API Reference pages.
- And to fix the URL shown in the cURL code.

#### What problem is this solving?

- Users were not able to import the openAPI schemas to Postman.
- The URL in the cURL code was missing a '/'.

#### How should this be manually tested?

Open any API Reference page, check if you are able to view and download the corresponding postman collection and check if when imported in Postman the collection content is correct.
